web/src/components/TaskHeader.test.tsx

```diff
@@ -249,7 +249,10 @@ describe("<TaskHeader />", () => {
         expect(currentUserData().profileData.dateOfFormation).toEqual(date.format("YYYY-MM-DD"))
       );
       expect(currentUserData().taskProgress[id]).toEqual("COMPLETED");
-      expect(mockPush).toHaveBeenCalledWith("/roadmap");
+      expect(mockPush).toHaveBeenCalledWith({
+        pathname: userData.profileData.businessPersona === "OWNING" ? "/dashboard" : "/roadmap",
```

Comment:
if you're specifying "STARTING" up above then this switch should not necessary right?

Reply:
why are we testing the query param logic in 2 places?